src/commands/flags.mjs

```diff
@@ -538,6 +538,50 @@ export const localBuildPath = {
   }
 }
 
+/** @type {CommandFlag} **/
+export const newAccountNumber = {
+  constName: 'newAccountNumber',
+  name: 'new-account-number',
+  definition: {
+    describe: 'new account number for node update transaction',
+    defaultValue: '',
+    type: 'string'
+  }
+}
+
+/** @type {CommandFlag} **/
+export const newAdminKey = {
+  constName: 'newAdminKey',
+  name: 'new-admin-key',
+  definition: {
+    describe: 'new admin key for the Hedera account',
+    defaultValue: '',
+    type: 'string'
+  }
+}
+
+/** @type {CommandFlag} **/
+export const gossipKey = {
+  constName: 'gossipKey',
+  name: 'gossip-key',
+  definition: {
+    describe: 'path and file name of the private signing gossip key in PEM key format to be used',
+    defaultValue: '',
+    type: 'string'
+  }
+}
+
+/** @type {CommandFlag} **/
+export const tlsKey = {
```

Comment:
make two:
- privateTlsKey
- publicTlsKey

Reply:
Updated with separate flags for public and private key